src/aui/framemanager.cpp

```diff
@@ -2211,6 +2213,11 @@ void wxAuiManager::GetDockSizeConstraint(double* width_pct, double* height_pct)
 
 void wxAuiManager::Update()
 {
+#ifdef __WXQT__
+    if ( !GetManagedWindow()->IsShownOnScreen() )
```

Comment:
My only explanation of your comment about this only working if `Update()` isn't it called in the sample is that `IsShownOnScreen()` doesn't work correctly in wxQt, could this be the case?

If so, ideal would be to make it work, of course, but failing that we could still use a flag and set it in `wxEVT_SHOW` handler, assuming this event is generated correctly.